pkg/validations/validations.go

```diff
@@ -53,12 +54,18 @@ func ValidateDomainNameFormat(dnsDomainName string) (int32, error) {
 }
 
 func ValidateHostname(name string) error {
+	if len(name) > maxHostNameLength {
```

Comment:
Isn't this redundant? (the regex should already check the limit)

Reply:
It is, good point! Removed it. Thanks!